packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -154,6 +154,7 @@ export default function FeedItemComponent({
   onAdClick,
   onReadArticleClick,
 }: FeedItemComponentProps): ReactElement {
+  const [justUpvoted, setJustUpvoted] = useState(false);
```

Comment:
With this being just a local state we will only be showing here if the user just upvoted, if we move away and come back this will go away, but it was mentioned we want to keep this until the user clicks.
I think it's fine this way as well as the post might not ever show again, but just checking